services/wheelmap/wheelmap.tester.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+const { colorScheme } = require('../test-helpers')
+const serverSecrets = require('../../lib/server-secrets')
+
+const t = (module.exports = require('../create-service-tester')())
+
+// If no Wheelmap token is provided (e.g. local development), API
+// responses are intercepted and mocked. If a token is found, the
+// requests will be forwarded to the real Wheelmap service.
+const noToken = !serverSecrets.wheelmap_token
```

Comment:
One thing I'm a little nervous about is tests failing silently when there's no token. What do you think about logging a warning when there's no token? If you do it here, it'll always print because the module is always loaded, though if you put it in a `before()` hook on each test, it'll only print when the tests are run.

Reply:
Oh yeah, what I did is indeed run regardless of the service, I'll fix that...